toolkit/docs/building/building.md

```diff
@@ -95,87 +103,48 @@ cd CBL-Mariner/toolkit
 
 # Checkout the desired release branch. The 1.0-stable tag tracks the most recent successful release of the 1.0 branch.
 git checkout 1.0-stable
-
-# Build the Go tools
-sudo make go-tools REBUILD_TOOLS=y
-
-# Get the package sources
-sudo make input-srpms DOWNLOAD_SRPMS=y
-```
-
-**NOTE: All subsequent commands are assumed to be executed from inside the toolkit directory.**
-
-### ISO Quick Start
-
-```bash
-# Build an ISO version of ./imageconfigs/core-efi.json entirely from downloaded, pre-built packages
-sudo make iso REBUILD_TOOLS=y REBUILD_PACKAGES=n
 ```
 
-### VHDX Quick Start
-
-```bash
-# Build a VHDX of ./imageconfigs/core-efi.json entirely from downloaded, pre-built packages
-sudo make image REBUILD_TOOLS=y REBUILD_PACKAGES=n
-```
-
-### Build from Sources Quick Start
-
-This is a **much slower** process which will download and compile sources rather than use pre-compiled packages.
-
-```bash
-# Build an image without downloading pre-compiled packages
-sudo make image REBUILD_TOOLS=y REBUILD_TOOLCHAIN=y DOWNLOAD_SRPMS=y PACKAGE_IGNORE_LIST="openjdk8 openjdk8_aarch64 shim-unsigned-aarch64" -j$(nproc)
-```
+**IMPORTANT:** The 1.0-stable tag always points to the latest known good build of CBL-Mariner. At this time, only the Mariner 1.0 branch is buildable.  This branch is continually updated with bug fixes, security vulnerability fixes or occassional feature enhancements.  Fixes may be applied to this branch at any time.  As those fixes are integrated into the branch the head of a branch may be temporarily unstable.  The 1.0-stable tag will remain fixed until the tip of the branch is validated and the latest source and binary packages (SRPMs and RPMs) are published.  At that point, the 1.0-stable tag is advanced.  
```

Comment:
Might be worth calling out `git fetch --tags` here to make sure you have the latest tags locally.

Reply:
added